packages/fx-core/resource/package.nls.json

```diff
@@ -235,6 +235,7 @@
   "core.generator.officeAddin.importProject.copyFiles": "Copying files...",
   "core.generator.officeAddin.importProject.convertProject": "Converting project...",
   "core.generator.officeAddin.importProject.updateManifest": "Modifying manifest...",
+  "core.generator.officeAddin.importOfficeProject.title": "Importing Existing Office Add-in Project",
```

Comment:
The title can't be in present continuous tense.
Proposed: Import Existing Office Add-in Project

Reply:
This title is used in the process of importing, so it here use importing instead of import. And it aligns with Outlook in 234 line. Thanks Supriya!